src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -944,7 +944,7 @@ def test_linux_webapp(self, resource_group):
             '.scm.azurewebsites.net/docker/hook'))
 
         result = self.cmd('webapp config container set -g {} -n {} --docker-custom-image-name {} --docker-registry-server-password {} --docker-registry-server-user {} --docker-registry-server-url {} --enable-app-service-storage {}'.format(
-            resource_group, webapp, 'foo-image', 'foo-password', 'foo-user', 'foo-url', 'false')).get_output_in_json()
```

Comment:
Instead changing the existing test to 'true' - I recommend updating the test with a new line to change the value from file to true & back & validate the changes work.

Reply:
Sure Sisira.

Reply:
Modified as requested.